Only in fvwm.mine: config.log
diff -r -u fvwm-2.5.5/fvwm/menus.c fvwm.mine/fvwm/menus.c
--- fvwm-2.5.5/fvwm/menus.c	Tue Oct 29 01:17:02 2002
+++ fvwm.mine/fvwm/menus.c	Fri Jan 10 04:37:24 2003
@@ -897,7 +897,7 @@
 
 	case XK_Left:
 	case XK_KP_4:
-	case XK_h: /* vi left */
+	case XK_j: /* vi left */
 		/* leave or enter a submenu */
 		saction = (MST_USE_LEFT_SUBMENUS(mr)) ? SA_ENTER : SA_LEAVE;
 		break;
@@ -933,7 +933,7 @@
 
 	case XK_Up:
 	case XK_KP_8:
-	case XK_k: /* vi up */
+	case XK_i: /* vi up */
 	case XK_p: /* prior */
 		if (fShiftedKey && !fControlKey && !fMetaKey)
 		{
@@ -954,7 +954,7 @@
 
 	case XK_Down:
 	case XK_KP_2:
-	case XK_j: /* vi down */
+	case XK_k: /* vi down */
 	case XK_n: /* next */
 		if (fShiftedKey && !fControlKey && !fMetaKey)
 		{
diff -r -u fvwm-2.5.5/libs/Target.c fvwm.mine/libs/Target.c
--- fvwm-2.5.5/libs/Target.c	Wed Sep  4 04:43:05 2002
+++ fvwm.mine/libs/Target.c	Fri Jan 10 04:37:25 2003
@@ -63,20 +63,20 @@
   {
   case XK_Up:
   case XK_KP_8:
-  case XK_k:
+  case XK_i:
   case XK_p:
     y_move = -y_move_size;
     break;
   case XK_Down:
   case XK_KP_2:
   case XK_n:
-  case XK_j:
+  case XK_k:
     y_move = y_move_size;
     break;
   case XK_Left:
   case XK_KP_4:
   case XK_b:
-  case XK_h:
+  case XK_j:
     x_move = -x_move_size;
     break;
   case XK_Right:
Only in fvwm.mine: tags
